larq_compute_engine/core/bitpack.h

```diff
@@ -11,25 +11,28 @@
 #include "larq_compute_engine/core/bitpack_aarch64.h"
 #endif
 
-// From @flatbuffers, used for the FLATBUFFERS_LITTLEENDIAN macro
-#include "flatbuffers/base.h"
+#include "flatbuffers/base.h"  // Used for the FLATBUFFERS_LITTLEENDIAN macro
+#include "ruy/check_macros.h"
+#include "ruy/profiler/instrumentation.h"
```

Comment:
I'd rather not put this in our core files but in the tflite kernel files, because of lce-micro where we use the same core files but don't have the ruy profiler. (The 'optimized' label in the aarch64 header would still be fine).
However I think that i saw some ruy profiler files in recent tf-micro builds so maybe this is not a problem anymore, I'll investigate.

Reply:
Apparently the file `ruy/profiler/instrumentation.h` is now pulled into the tf-micro builds, maybe because they realized these tags were scattered all over the code so it was easier to just add that profiler file.
That is the *only* file, however, so the `check_macros.h` file can definitely not be used.
We could just move the tags into the tflite kernel files, except that we can then not see if the optimized or non-optimized path is used. I'm also fine with leaving the tags here (given that we can get rid of `check_macros.h`). What do you think @adamh ?

Reply:
Awesome, I've removed the `check_macros.h` import and replaced it with `tflite/kernels/internal/types.h` which I think should be fine for micro?